--- gtk+-2.0.vapi.orig	2011-01-30 14:36:09.783843890 +0100
+++ gtk+-2.0.vapi	2011-01-30 14:37:14.007344239 +0100
@@ -5729,7 +5729,7 @@
 		public abstract unowned string get_chars (int start_pos, int end_pos);
 		public bool get_editable ();
 		public abstract int get_position ();
-		public abstract bool get_selection_bounds (int start_pos, int end_pos);
+		public abstract bool get_selection_bounds (out int start_pos, out int end_pos);
 		public void paste_clipboard ();
 		[CCode (vfunc_name = "set_selection_bounds")]
 		public abstract void select_region (int start_pos, int end_pos);
@@ -7153,7 +7153,7 @@
 	[CCode (cheader_filename = "gtk/gtk.h")]
 	public delegate void ClipboardTargetsReceivedFunc (Gtk.Clipboard clipboard, out Gdk.Atom atoms, int n_atoms);
 	[CCode (cheader_filename = "gtk/gtk.h")]
-	public delegate void ClipboardTextReceivedFunc (Gtk.Clipboard clipboard, string text);
+	public delegate void ClipboardTextReceivedFunc (Gtk.Clipboard clipboard, string? text);
 	[CCode (cheader_filename = "gtk/gtk.h")]
 	public delegate void ClipboardURIReceivedFunc (Gtk.Clipboard clipboard, string uris);
 	[CCode (cheader_filename = "gtk/gtk.h", has_target = false)]
