--- gtk+-3.0.vapi.orig	2011-08-07 16:38:45.800906814 +0200
+++ gtk+-3.0.vapi	2011-08-07 19:21:51.100391281 +0200
@@ -1852,7 +1852,11 @@
 		public void add_pattern (string pattern);
 		public void add_pixbuf_formats ();
 		public bool filter (Gtk.FileFilterInfo filter_info);
+		[CCode (cname = "gtk_file_filter_get_name")]
+		public unowned string get_filter_name ();
 		public Gtk.FileFilterFlags get_needed ();
+		[CCode (cname = "gtk_file_filter_set_name")]
+		public void set_filter_name (string name);
 	}
 	[Compact]
 	[CCode (cheader_filename = "gtk/gtk.h")]
